pandas/core/construction.py

```diff
@@ -498,6 +498,8 @@ def sanitize_array(
     """
     if isinstance(data, ma.MaskedArray):
         data = sanitize_masked_array(data)
+    elif isinstance(data, np.matrix):
```

Comment:
np.matrix is a subclass of np.ndarray, so can this check go inside the `if isinstance(data, np.ndarray)` below?

Reply:
:+1: